ui/craco.config.js

```diff
@@ -33,6 +33,17 @@ function withFilterWarningsPluginForCssImportOrderConflict({ webpackConfig }) {
   return webpackConfig;
 }
 
+// to workaround https://github.com/DocSpring/craco-less/issues/34
+function appendLoaderOptionsIntoUseForSassRule({ webpackConfig }) {
+  const oneOfRule = webpackConfig.module.rules.find(rule => rule.oneOf);
+  const sassRule = oneOfRule.oneOf.find(
+    rule => rule.test && rule.test.toString().includes("scss|sass")
+  );
+  sassRule.use = [...(sassRule.use || []), ...sassRule.loader];
```

Comment:
```.loader`` can be ``undefined``?

Reply:
shouldn't be unless we change our deps like `react-scripts`